PackageInfo.g

```diff
@@ -110,13 +111,26 @@ Keywords := [ "Automatic documentation, GAP, GAPDoc" ],
 AutoDoc := rec(
     TitlePage := rec(
         Copyright := Concatenation(
-                    "&copyright; 2012-2014 by Sebastian Gutsche and Max Horn<P/>\n\n",
-                    "This package may be distributed under the terms and conditions of the\n",
-                    "GNU Public License Version 2.\n"
-                ),
+            "&copyright; 2012-2014 by Sebastian Gutsche and Max Horn<P/>\n\n",
```

Comment:
Should be 2017

Reply:
Also, should we probably use the `"""` syntax here? Would be a good place to advertise it.

Reply:
2017 done.  Leaving """ to you!